webapps/hello-servlet-6/pom.xml

```diff
@@ -6,7 +6,7 @@
     <artifactId>jetty-webapp-examples</artifactId>
     <version>12.0.x</version>
   </parent>
-  <artifactId>hello-servlet-5</artifactId>
```

Comment:
Please do not change the `hello-servlet-5`.

I have no problem with introducing a new `hello-servlet-6` tho.

Reply:
I have reverted servlet 6 as you have suggested